CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,49 @@
+# Contributor Code of Conduct
```

Comment:
I think we should reference here the general GraphQL code of conduct and not introduce a new one.

Reply:
Good idea, I'll reference https://graphql.org/codeofconduct/